.eslintrc.json

```diff
@@ -0,0 +1,67 @@
+{
+  "parser": "@typescript-eslint/parser", // Specifies the ESLint parser
+  "env": {
+    "browser": true
+  },
+  "settings": {
+    "react": {
+      "version": "detect" // Tells eslint-plugin-react to automatically detect the version of React to use
+    }
+  },
+  "root": true,
+  "parserOptions": {
+    "ecmaVersion": "latest", // Allows for the parsing of modern ECMAScript features
+    "sourceType": "module" // Allows for the use of imports
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:import/recommended",
+    "plugin:import/typescript",
+    "plugin:jsx-a11y/recommended",
+    "plugin:react/recommended",
+    "plugin:react/jsx-runtime",
+    "plugin:react-hooks/recommended",
+    "react-app",
+    "prettier"
+  ],
+  "rules": {
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      { "argsIgnorePattern": "^_" }
+    ],
+    "sort-imports": [
+      "error",
+      {
+        "ignoreDeclarationSort": true
+      }
+    ],
+    "import/order": [
+      "error",
+      {
+        "newlines-between": "never"
+      }
+    ],
+    "react-hooks/rules-of-hooks": "error", // Checks rules of Hooks
+    "react-hooks/exhaustive-deps": "warn" // Checks effect dependencies
+  },
+  "overrides": [
+    // Test files.
+    {
+      "files": ["./**/*.spec.*", "./**/*.test.*"],
+      "extends": [
+        "plugin:cypress/recommended",
+        "plugin:jest/recommended",
+        "plugin:testing-library/react"
+      ],
+      "env": {
+        "cypress/globals": true
+      }
+    },
+    // Stories.
+    {
+      "files": ["./**/*.story.*", "./**/*.stories.*"],
+      "extends": ["plugin:storybook/recommended"]
+    }
+  ]
+}
```

Comment:
Good point, I believe if we don't specify a configuration file it will default to whatever it can find. I could open a PR for that but wouldn't know how to test it.

Reply:
Ah true, I don't remember if it really did need to be specified—but I suspect you're right. I'll look into it.